firebase-firestore/src/main/java/com/google/firebase/firestore/util/AsyncQueue.java

```diff
@@ -74,7 +74,13 @@
      * A timer used to retry transactions. Since there can be multiple concurrent transactions,
      * multiple of these may be in the queue at a given time.
      */
-    RETRY_TRANSACTION
+    RETRY_TRANSACTION,
+    /**
+     * A timer used in RemoteStore to monitor when a connection attempt is unsuccessful and retry
+     * accordingly. While `ONLINE_STATE_TIMEOUT` is used to transition from UNKNOWN to OFFLINE,
+     * `CONNECTIVITY_ATTEMPT_TIMER` is used for tracking and retrying connectivity attempts.
+     */
+    CONNECTIVITY_ATTEMPT_TIMER
```

Comment:
stale

Reply:
still stale

Reply:
What needs to change here? I thought I updated it to reflect that this time is for gRPC attempts.

Reply:
Oops!  Sorry.  I thought this was something else.  My bad.